docs/sdk/v3/guides/04-routing.md

```diff
@@ -3,4 +3,51 @@ id: routing
 title: Routing a Swap
 ---
 
-TODO
\ No newline at end of file
+## Introduction
+
+This guide will cover how to use the Uniswap Autorouter to compute optimal routes and execute swaps.
+It is based on the [Routing code example](https://github.com/Uniswap/examples/tree/main/v3-sdk/routing), found in the Uniswap code examples [repository](https://github.com/Uniswap/examples).
+To run this example, check out the guide's [README](https://github.com/Uniswap/examples/blob/main/v3-sdk/routing/README.md) and follow the setup instructions.
```

Comment:
Thoughts on also doing a 1 sentence briefer on what routing/smart-order-routing is as opposed to *trading* to drive the distinction home?